mpisppy/convergers/norm_rho_converger.py

```diff
@@ -12,9 +12,6 @@
 class NormRhoConverger(mpisppy.convergers.converger.Converger):
 
     def __init__(self, ph):
-        ## TODO: this will never do anything unless the norm rho updater is also used
-        ## TODO: the rho updater should leave a flag on the data block of the local subproblems to indicate
-        ##       its use and this converger should check for it.
         if 'nrom_rho_converger_options' in ph.options and \
```

Comment:
```suggestion
        if 'norm_rho_converger_options' in ph.options and \
```
Might as well fix this typo.